streamz/sources.py

```diff
@@ -57,13 +56,16 @@ class from_textfile(Source):
     -------
     Stream
     """
-    def __init__(self, f, poll_interval=0.100, **kwargs):
+    def __init__(self, f, poll_interval=0.100, asynchronous=True, **kwargs):
```

Comment:
We probably can't merge with this as the default.  I'll try to find some time in the next few days to figure out a solution here.

Reply:
Note that I added it to from_textfiles too, for now.